schemas/context/identity.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/identity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Identity",
+  "type": "object",
+  "description": "Identity from identity services which is the fact of being who or what a person or thing is.\n",
+  "definitions": {
+    "identity": {
+      "properties": {
+        "@id": {
```

Comment:
You know, I'm a stickler for `uri`s in all places. But here I wonder if it is really practical, given that we cannot control what an external service is using for IDs. In some cases, it might be numbers, in some UUIDs, in some random strings. I think `string` is the safest bet, which means we should also rename the property to just `xdm:id`.

Reply:
I can agree with that.  @cdegroot-adobe had a similar comment.